modules/REST/V1/Languages.php

```diff
@@ -0,0 +1,803 @@
+			);
+		}
+
+		// At this point, `$request['locale']` is provided (but maybe not valid).
+		$prepared = $this->prepare_item_for_database( $request );
+
+		if ( is_wp_error( $prepared ) ) {
+			return $prepared;
+		}
+
+		/**
+		 * @phpstan-var object{
+		 *    locale: non-empty-string,
+		 *    slug: non-empty-string,
+		 *    name: non-empty-string,
+		 *    rtl: bool,
+		 *    term_group: int,
+		 *    flag: non-empty-string,
+		 *    no_default_cat: bool
+		 * } $prepared
+		 * @phpstan-var array{
+		 *    locale: non-empty-string,
+		 *    slug: non-empty-string,
+		 *    name: non-empty-string,
+		 *    rtl: bool,
+		 *    term_group: int,
+		 *    flag: non-empty-string,
+		 *    no_default_cat: bool
+		 * } $args
+		 */
+		$args   = (array) $prepared;
+		$result = $this->languages->add( $args );
+
+		if ( is_wp_error( $result ) ) {
+			return $this->add_status_to_error( $result );
+		}
+
+		/** @var PLL_Language */
+		$language = $this->languages->get( $prepared->locale );
+		return $this->prepare_item_for_response( $language, $request );
+	}
+
+	/**
+	 * Retrieves one language from the collection.
+	 *
+	 * @since 3.7
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 *
+	 * @phpstan-template T of array
+	 * @phpstan-param WP_REST_Request<T> $request
+	 */
+	public function get_item( $request ) {
+		$language = $this->get_language( $request );
+
+		if ( is_wp_error( $language ) ) {
+			return $language;
+		}
+
+		return $this->prepare_item_for_response( $language, $request );
+	}
+
+	/**
+	 * Updates one language from the collection.
+	 *
+	 * @since 3.7
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 *
+	 * @phpstan-template T of array
+	 * @phpstan-param WP_REST_Request<T> $request
+	 */
+	public function update_item( $request ) {
+		$prepared = $this->prepare_item_for_database( $request );
+
+		if ( is_wp_error( $prepared ) ) {
+			// Should not happen, but if it does, it's our fault.
+			return $prepared;
+		}
+
+		/**
+		 * @phpstan-var object{
+		 *     lang_id: int,
+		 *     locale: non-empty-string,
+		 *     slug: non-empty-string,
+		 *     name: non-empty-string,
+		 *     rtl: bool,
+		 *     term_group: int,
+		 *     flag?: non-empty-string
+		 * } $prepared
+		 * @phpstan-var array{
+		 *     lang_id: int,
+		 *     locale: non-empty-string,
+		 *     slug: non-empty-string,
+		 *     name: non-empty-string,
+		 *     rtl: bool,
+		 *     term_group: int,
+		 *     flag?: non-empty-string
+		 * } $args
+		 */
+		$args   = (array) $prepared;
+		$update = $this->languages->update( $args );
+
+		if ( is_wp_error( $update ) ) {
+			return $this->add_status_to_error( $update );
+		}
+
+		/** @var PLL_Language */
+		$language = $this->languages->get( $prepared->lang_id );
```

Comment:
With the same objective as above:
```suggestion
		$language = $this->languages->get( $args['lang_id'] );
```

Reply:
Done in [c860600](https://github.com/polylang/polylang/pull/1505/commits/c86060005637a400d23e9a516817dfa8bb3e84f4).